styles/main.css

```diff
@@ -0,0 +1,184 @@
+@import url("https://fonts.googleapis.com/css?family=Open+Sans|Pacifico");
```

Comment:
awesome job importing fonts from Google's API! I remember this feeling a bit "tricky" the first time. ;)